students/swietczak_monika/lesson_06_dicts_tuples_sets_args_kwargs/uppercase.py

```diff
@@ -0,0 +1,9 @@
+def capitalize(lower_case_word):
+    lower_list = list(lower_case_word)
+    lower_list[0] = chr(ord(lower_list[0]) - 32)
```

Comment:
It's a bit complicated solution.
 
Please consider the following
```
 title = lower_case_word [0].upper() + lower_case_word [1:]
or use dedicated method for this operation
lower_case_word.title()
```

Reply:
OK I changed it, I wrote it such way because it has been suggested so in Snakify.